tests/data/author/controls/control_with_components_and_params.md

```diff
@@ -10,7 +10,8 @@ x-trestle-fedramp-props:
     - id: admin-unix
     - id: ux-coder
 x-trestle-set-params:
-  ac-1_prm_3: orig val in control md header
+  ac-1_prm_3:
+    values: orig val in control md header
```

Comment:
Are there tests for the headers with list of `values` and choices?

Reply:
there are tests for the param->dict conversions back and forth including multiple values and validity checks in test_param_to_dict()  I will check if the higher level tests exercise it.